src/styles/_Navbar.scss

```diff
@@ -1,13 +1,18 @@
 .Navbar {
+  background-image: linear-gradient(#5b5b5b, #2b3034);
+
+}
+
+.main-nav {
   display: flex;
   flex-wrap: wrap;
   justify-content: space-between;
   min-height: 75px;
-  background-image: linear-gradient(#5b5b5b, #2b3034);
 }
 
 .title {
   color: #ff1818;
+  letter-spacing: 3.5px;
```

Comment:
Makes the GAMEFLIX title look a little cleaner IMO, up for change back if we want.

Reply:
Oh nice, adding this to the homepage as well!